src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -302,14 +300,12 @@ export const Sidebar = React.forwardRef<HTMLDivElement, Props>(
                   refetch={() => refetchAll()}
                 />
               </Container>
-              {market === Market.Se && (
-                <DetailsModal
-                  offerQuote={offerData.quotes[0]}
-                  refetch={refetchAll}
-                  isVisible={detailsModalIsOpen}
-                  onClose={() => setDetailsModalIsOpen(false)}
-                />
-              )}
+              <DetailsModal
+                offerQuote={offerData.quotes[0]}
```

Comment:
I am not sure about how it should work with bundles. I think that this might be a different pr when we don't have the complete design?  But in any case I would need a debrief 🤓

Reply:
Sounds wise, but lets disable this for norway in that case, just until we handle that :) so we don't risk getting one quote with 1 co-insured and another with 2, for example :)

Reply:
Ah good catch, didn't think about how to treat bundles either. We might need to do so visual clarification as well so the user knows what they are editing